worker/src/main/java/gov/cms/ab2d/worker/processor/ContractData.java

```diff
@@ -30,4 +38,12 @@ public boolean remainingRequestHandles() {
     public FhirVersion getFhirVersion() {
         return job.getFhirVersion();
     }
+
+    public int getNumberBenes(Future<ProgressTrackerUpdate> agg) {
+        Integer number = hashBeneMapper.get(agg.hashCode());
+        if (number == null) {
```

Comment:
Pretty sure there is a get or default on the hash mapper, if this really is necessary.

Reply:
Yes, changed it.